### ISSUE TITLE
@aws-amplify/api has a vulnerable dependency, Axios:0.17.0

### ISSUE THREAD

**Issue by mrowe009** [aws-amplify/amplify-cli#1573]
@aws-amplify/api current version has a vulnerable dependency Axios: 0.17.0

Vulnerable versions: <= 0.18.0
Patched version: 0.19.0

Axios up to and including 0.18.0 allows attackers to cause a denial of service (application crash) by continuing to accepting content after maxContentLength is exceeded.

upgrading to 0.19.0 fixes the issue.

**Comment by mrowe009** [comment]
I'm closing this issue as aws-amplify is goign to resolve this, and i was told to put up the issue here. Sorry for any confusion.

**Comment by ozbillwang** [comment]
get the same. 

```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Denial of Service                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ axios                                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=0.18.1                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ aws-amplify                                                  │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ aws-amplify > @aws-amplify/api > axios                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/880                             │
└───────────────┴──────────────────────────────────────────────────────────────┘
```
@mrowe009 

Where can I get update for this issue.

**Comment by mrowe009** [comment]
@ozbillwang follow this thread, as the issue mainly pertains to this package https://github.com/aws-amplify/amplify-js/issues/3394#event-2388897308
